op-guide/prometheus.md

```diff
@@ -0,0 +1,100 @@
+# Use Prometheus and Grafana in TiDB
+
+This page shows you how to deploy Prometheus and Grafana.  
+## Preparation
+Before you start, make sure you have: 
+
++ Installed the [Pushgateway](https://github.com/prometheus/pushgateway) and [Prometheus](https://prometheus.io/docs/introduction/install)
++ Started the TiDB/TiKV/PD with [metrics-addr](https://github.com/pingcap/docs/blob/master/op-guide/configuration.md)
+
+## Step 1. Configure the Prometheus
+
+See the following `prometheus.yml` as an example:
+
+```
+#global config
+global:
+  scrape_interval:     15s # By default, scrape targets every 15 seconds.
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+
+# scrape_configs:
+# The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+#  - job_name: 'prometheus'
+
+#    Override the global default and scrape targets from this job every 5 seconds.
+#    scrape_interval: 5s
+#    static_configs:
+#      - targets: ['localhost:9090']
+
+scrape_configs:
+  - job_name: "cluster"
+
+    # Override the global default and scrape targets from this job every 5 seconds.
+    # scrape_interval: "5s"
+
+    # honor_labels controls how Prometheus handles conflicts between labels that are
+    # already present in scraped data and labels that Prometheus would attach
+    # server-side ("job" and "instance" labels, manually configured target
+    # labels, and labels generated by service discovery implementations).
+    #
+    # If honor_labels is set to "true", label conflicts are resolved by keeping label
+    # values from the scraped data and ignoring the conflicting server-side labels.
+    #
+    # If honor_labels is set to "false", label conflicts are resolved by renaming
+    # conflicting labels in the scraped data to "exported_<original-label>" (for
+    # example "exported_instance", "exported_job") and then attaching server-side
+    # labels. This is useful for use cases such as federation, where all labels
+    # specified in the target should be preserved.
+    
+    honor_labels: true
+
+    # metrics_path defaults to '/metrics'
+    # scheme defaults to 'http'.
+    # here target used pushgateway address.
+    static_configs:
+      - targets: ['192.168.199.118:9091'] 
+
+```
+
+**Note**:
+ - If you use pushgateway, set `honor_labels:true`.
+ - If you use Docker start TiDB, add the `-h $HOSTNAME` option to define the hostname in Docker. Then, you can start Prometheus using the `prometheus.yml` file.
```

Comment:
If we use k8s later, we may not get the hostname easily, so I don't think it is necessary to pass hostname here. 
We can use other values for the `instance` field. 
+ For TiDB, We can use host:port 
+ For PD, we can use PD name or the first advertise-url address 
+ For TiKV, we can use advertise-addr 

/cc @shenli @huachaohuang @overvenus